script.js

```diff
@@ -0,0 +1,14 @@
+$(function() {
+	var carouselList = $('#carousel ul');
+	setInterval(changeSlides, 1000);
+	$(changeSlides() {
+		carouselList.animate({'marginLeft':-400}, 500, moveFirstSlide);
+	});
+});
+
+$(moveFirstSlide() {
```

Comment:
this also should be added inside function checking loading DOM. same as above, `$` unnecessary, and `moveFirstSlide` should be declared as function :)